messages/compiler/cmfc.py

```diff
@@ -35,6 +35,7 @@ def translate(ast, language, namespace, output):
         print("Generating Python source code")
         from python import pygen
         code = pygen.translate(ast, namespace)
+        print(output)
```

Comment:
That seems arbitrary?! Why do we print here? Also, there is a print statement right above.